dashboard/app/models/user.rb

```diff
@@ -120,6 +122,9 @@ class User < ApplicationRecord
     section_attempts_last_reset
     share_teacher_email_regional_partner_opt_in
     last_verified_captcha_at
+    gender_student_input
```

Comment:
Maybe a nitpick, but should this be `gender_user_input` just in case we allow teacher accounts to also share their gender in the future?